src/main/groovy/edu/oregonstate/mist/locations/MergeUtil.groovy

```diff
@@ -162,5 +163,17 @@ class MergeUtil {
             }
         }
 
+        def missingLocations = []
+        campusMapData.each { key, value ->
+            if(!data.find { it.id == value.id }) {
+                missingLocations.add(value.id)
+            }
+        }
+        float ratioMissing = missingLocations.size() / campusMapData.size()
```

Comment:
Definitely a good idea. I think this might be helpful in determining what the threshold should be in the first place. For example, when I was testing without any facilLocation data, ratioMissing was still ~0.87 which is fairly high for having 0 facilLocations. Maybe playing around with what we give it will help us find this value.